instrumentation/redis/README.md

```diff
@@ -47,8 +47,8 @@ end
 OpenTelemetry::SDK.configure do |c|
   c.use 'OpenTelemetry::Instrumentation::Redis', {
     # The obfuscation of arguments in the db.statement attribute is enabled by default.
-    # To disable, set enable_statement_obfuscation to false.
-    enable_statement_obfuscation: true,
+    # To disable, set db_statement to :include or :omit if the tag not required.
```

Comment:
Obfuscate is also a valid and supported option.
```suggestion
    # To disable, set db_statement to :include, :obfuscate or :omit if the tag not required.
```

Reply:
A bit rephrased the comment - I split options to separate lines.